src/common/System.cpp

```diff
@@ -335,8 +337,10 @@ static void CrashHandler(const void* data, size_t n)
 
 void SetupCrashHandler()
 {
+#if !defined(__saigo__)
```

Comment:
Why is this removed? It's the only debugging mechanism we have for NaCl.

Reply:
Probably because I didn't make it work.